### ISSUE TITLE
CSV export, please add ISO 639-3 codes

### ISSUE THREAD

**Issue by nthieberger** [paradisec-archive/nabu#656]
Please add the codes to the export table as well as the names. It is reasonably urgent as we have to supply metadata with these codes to a funding body. HOWEVER! If this is addressed by the new GraphQL API then ignore this request. But please let me know how to do it:)

**Comment by j03w** [comment]
@nthieberger yes it does actually returned through the new GraphQL API already. I'll get back to you with the sample query to fetch that tomorrow arvo.

**Comment by JonoGillett** [comment]
@nthieberger here is an example query which will return the code and name for subject and content languages from GraphQL. Currently you will need to follow the pagination to retrieve all records.

GraphQL is available in staging http://115.146.80.168/graphiql

```
{
  items(limit: 500, page: 1) {
    total
    next_page
    results {
      subject_languages {
        code
        name
      }
      content_languages {
        code
        name
      }
    }
  }
}
```

If you only need the languages for a certain collection you could use `items(full_identifier: "AA1", page: 1, limit: 500) {......}`
